blog/_posts/2024-05-14-core-extension.md

```diff
@@ -0,0 +1,87 @@
+- **Extensions**: Managed via the Extension API (`Microsoft.KubernetesConfiguration/extensions`), supporting both AKS and [Arc-enabled Kubernetes environments](https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/overview).
+
+### What Are Core Kubernetes Extensions?
+
+ Core Kubernetes extensions are a Kubernetes extension that provides broader region availability, a native AKS experience, safer version management, enhanced security, efficiency, and reliability on AKS. Important AKS capabilities can be supported through core Kubernetes extensions while maintaining seamless integration with AKS just like add-ons. This approach also paves the way for more services and applications to be supported across both AKS and Arc-connected Kubernetes environments in the future.
+
+#### Broader region availability
+Core Kubernetes extensions are available in all regions across Azure Public cloud, Azure Government and China regions. See the full list of regions at [Azure Arc enabled Kubernetes region support](https://azure.microsoft.com/en-us/explore/global-infrastructure/products-by-region/?products=azure-arc&regions=all).
+
+#### Native AKS experience
+* Add-ons migrated to core extensions: Some AKS add-ons, such as Container Insights, Azure Managed Prometheus, and App Monitoring, are transitioning to core extensions. Although the underlying technology shifts to extensions, the user experience remains unchanged. The CLI command to enable/disable migrated add-ons are the same as before migration.
+
+    ```bash
+    az aks create/update --resource-group <group> --name <cluster> --enable/disable <add-on>
+    ```
+* Regular extensions graduated to core extensions: Key extensions like Azure Backup are also being upgrading to core extensions for better integration with AKS.
+
+    CLI commands:
+    * Create/delete a regular extension:
+    ```bash
+    az k8s-extension create/delete --extension-type <type> --resource-group <group> --cluster-name <name> --cluster-type <clusterType> --name <extension name>
+    ```
+    * Create/delete a core extension:
+    ```bash
+    az aks extension create/delete --extension-type <type> --resource-group <group> --cluster-name <name> --name <core extension name>
+    ```
+
+#### Safer version management
+In principle, core extensions follow the same update rules as add-ons: patch versions can be upgraded within a Kubernetes minor version, while major/minor upgrades occur only with Kubernetes minor version updates to avoid introducing breaking changes in a Kubernetes version. Exceptions are allowed if the extension has proven testing and safeguards against unexpected breaking changes in new minor/major versions.
+
+#### Improved security, efficiency and reliability
+
+ExtensionManager is moved from user node pool to AKS Control Plane, which improved extension security, efficiency, and reliability.
```

Comment:
Some small edits"ExtensionManager is moved from _the_ user node pool to _the_ AKS Control Plane, which _improves_ extension security, efficiency, and reliability."